redis/tests/test_async.rs

```diff
@@ -97,6 +97,29 @@ fn test_pipeline_transaction() {
     .unwrap();
 }
 
+#[test]
+fn test_push() {
```

Comment:
Could we give this and the other `test_push` test more descriptive names? I'm only 30 confident sure I know what this test is testing :sweat_smile:

Reply:
:D u are right, is it ok now ? i don't think parse tests needs more descriptive names.